content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -97,31 +104,17 @@ You can also configure `serverless deploy --stage local` as a `deploy` script in
 }
 ```
 
-You can give the above setup a run by deploying your Serverless project and checking the Lambda function that has been created via our existing Serverless project:
+With this configuration, you can now run the deployment script using:
 
 {{< command >}}
 $ npm run deploy
-$ awslocal lambda list-functions
-
-{
-    "Functions": [
-        {
-            "FunctionName": "serverless-elb-local-hello",
-            "FunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:serverless-elb-local-hello",
-            "Runtime": "nodejs12.x",
-            "Role": "arn:aws:iam::000000000000:role/serverless-elb-local-us-east-1-lambdaRole",
-            "Handler": "handler.hello",
-            ...
-        }
-    ]
-}
 {{< / command >}}
 
-We can now create our target Lambda functions to work with Node.js runtime and configure ELB Application Load Balancers via our `serverless.yml` configuration.
+This will execute the `serverless deploy --stage local` command, deploying your Serverless project to LocalStack.
 
 ## Create Lambda functions & ELB Application Load Balancers
 
-Let us now create two Lambda functions, named `hello1` and `hello2`, which can run over Node.js 12.x runtime. Open `handler.js` and replace the existing code with the following:
+Now, let's create two Lambda functions named `hello1` and `hello2` that will run on the Node.js 12.x runtime. Open the `handler.js` file and replace the existing code with the following:
```

Comment:
I'm following the tutorial and this point I do not have a `handler.js` file yet (I also do not now what the "existing code" should be). Could you please clarify 🙏

Reply:
While running the `serverless create --template aws-nodejs --path serverless-elb` command in a previous step, a `handler.js` file was generated.

Reply:
ah, sorry, totally missed that step!